tests/app/main/views/test_dashboard.py

```diff
@@ -683,17 +683,29 @@ def test_should_show_monthly_breakdown_of_template_usage(
         "main.template_usage", service_id=SERVICE_ONE_ID, **extra_args
     )
 
-    mock_get_monthly_template_usage.assert_called_once_with(SERVICE_ONE_ID, 2016)
+    mock_get_monthly_template_usage.assert_called_once_with(SERVICE_ONE_ID, 2017)
 
     table_rows = page.select("tbody tr")
 
     assert " ".join(table_rows[0].text.split()) == (
         "My first template " "Text message template " "2"
     )
 
-    assert len(table_rows) == len(["October"])
+    assert len(table_rows) == len(["January"])
     assert len(page.select(".table-no-data")) == len(
-        ["November", "December", "January"]
+        [
+            "January",
+            "February",
+            "March",
+            "April",
+            "May",
+            "June",
+            "July",
+            "August",
+            "September",
+            "October",
+            "November",
```

Comment:
Do we need December in here, too?

```suggestion
            "November",
            "December",
```

Reply:
I believe so. Let me run the tests and see what I potentially missed here.